tests/test_upstream_versions.py

```diff
@@ -1943,6 +1945,46 @@ def test_latest_version_cratesio(tmpdir):
         assert ver == attempt["new_version"]
 
 
+@pytest.mark.parametrize(
+    "yaml_path",
+    [
+        pytest.param(YAML_PATH, id="meta.yaml"),
+        pytest.param(YAML_PATH_V1, id="recipe.yaml"),
+    ],
+)
+def test_latest_version_gittags(tmpdir, yaml_path):
+    name = "libtirpc"
+    recipe_path = os.path.join(yaml_path, "libtirpc-gittags.yaml")
+    curr_ver = "1.3.6"
+    new_ver = "1.3.7"
+    source = GitTags()
+
+    with open(recipe_path) as fd:
+        inp = fd.read()
+
+    pmy = LazyJson(os.path.join(str(tmpdir), "cf-scripts-test.json"))
+    with pmy as _pmy:
+        yml = (
+            parse_recipe_yaml(inp, use_container=False)
+            if yaml_path == YAML_PATH_V1
+            else parse_meta_yaml(inp, use_container=False)
+        )
+        _pmy.update(yml["source"])
```

Comment:
The tests is passing when it should not because this line is adding more keys to the node attributes than should be there. We should only add the url and the hash keys here.

Reply:
I am not sure. Let's see what happens!